### PR TITLE
Prompt before removing yourself from principals

### PR DESCRIPTION
**Pull request by mic4ael**

### CODE REVIEW

File: indico/htdocs/js/indico/Core/Dialogs/Users.js [hunk 1]
```diff
@@ -1338,11 +1338,26 @@ type("UserListWidget", ["ListWidget"],
...
+
+                function setResult(result) {
                     if (result) {
                         self.set(user.key, null);
                         self.userListField.inform();
                     }
+                }
+
+                if (currentUserId === userId) {
+                    confirmed = confirmPrompt($T.gettext('Are you sure you want to remove yourself from the list?'),
```
Comment (ThiefMaster) [inline indico/htdocs/js/indico/Core/Dialogs/Users.js]: I would explain that you may lose access unless you are a manager of the category.

Comment (ThiefMaster) [inline indico/htdocs/js/indico/Core/Dialogs/Users.js]: And this should only apply to management lists (event/category), so it should probably be configurable in the WTForms field level

Comment (mic4ael) [inline indico/htdocs/js/indico/Core/Dialogs/Users.js]: As we discussed I think it is a good thing to have it on every field using `PrincipalListField` internally (not only for ACLs). But if you insist on having it only for ACLs, I will change it.

Comment (ThiefMaster) [inline indico/htdocs/js/indico/Core/Dialogs/Users.js]: the comment was from before we talked about it ;) we can keep it like this for now

Comment (mic4ael) [inline indico/htdocs/js/indico/Core/Dialogs/Users.js]: so maybe it can be merged? :)

**Pull request merged**